sdk/schemaregistry/Azure.Data.SchemaRegistry/src/Generated/Models/SchemaId.cs

```diff
@@ -5,21 +5,15 @@
 
 #nullable disable
 
-using System;
-using Azure.Data.SchemaRegistry;
-
 namespace Azure.Data.SchemaRegistry.Models
 {
     /// <summary> Object received from the registry containing schema identifiers. </summary>
     internal readonly partial struct SchemaId
     {
         /// <summary> Initializes a new instance of <see cref="SchemaId"/>. </summary>
         /// <param name="id"> Schema ID that uniquely identifies a schema in the registry namespace. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="id"/> is null. </exception>
         internal SchemaId(string id)
```

Comment:
`SchemaId.Id` property is optional. Also, type is internal and isn't used anywhere.